pkg/apis/troubleshoot/v1beta2/analyzer_shared.go

```diff
@@ -58,6 +58,18 @@ type DeploymentStatus struct {
 	Name        string     `json:"name" yaml:"name"`
 }
 
+type ClusterResource struct {
+	AnalyzeMeta   `json:",inline" yaml:",inline"`
+	CollectorName string     `json:"collectorName,omitempty" yaml:"collectorName,omitempty"`
+	Outcomes      []*Outcome `json:"outcomes" yaml:"outcomes"`
+	Kind          string     `json:"kind" yaml:"kind"`
+	ClusterScoped bool       `json:"clusterScoped" yaml:"clusterScoped"`
+	Namespace     string     `json:"namespace,omitempty" yaml:"namespace,omitempty"`
+	Name          string     `json:"name" yaml:"name"`
+	YamlPath      string     `json:"yamlPath" yaml:"yamlPath"`
+	Value         string     `json:"value,omitempty" yaml:"value,omitempty"`
```

Comment:
I wonder if this would be clearer if it was `expectedValue` instead of `value`. Makes it more intentional in the analyzer code what you're testing/looking for.

Reply:
Yeh that's a point.